tools/FileTestDefs.sh

```diff
@@ -418,6 +418,35 @@ pragma_once_test() {
 }
 standard_checks+=(pragma_once)
 
+# Check for 'return Py_None;' in all C++ files
+py_return_none() {
+    is_c++ "$1" && \
+        whitelist "$1" && \
+        staged_grep -q -x '.*return Py_None;.*' "$1"
+}
+py_return_none_report() {
+    echo "Did not find 'return Py_None;' in these files:"
```

Comment:
This should probably say the opposite :) and could suggest to use `Py_RETURN_NONE` instead